sentinel.conf

```diff
@@ -124,6 +124,124 @@ sentinel monitor mymaster 127.0.0.1 6379 2
+#                           disabled command. Note that this form is not
+#                           allowed as negative like -SENTINEL|FAILOVER, but
+#                           only additive starting with "+".
+#  allcommands  Alias for +@all. Note that it implies the ability to execute
+#               all the future commands loaded via the modules system.
+#  nocommands   Alias for -@all.
+#  ~<pattern>   Add a pattern of keys that can be mentioned as part of
+#               commands. For instance ~* allows all the keys. The pattern
+#               is a glob-style pattern like the one of KEYS.
+#               It is possible to specify multiple patterns.
+#  allkeys      Alias for ~*
+#  resetkeys    Flush the list of allowed keys patterns.
+#  ><password>  Add this password to the list of valid password for the user.
+#               For example >mypass will add "mypass" to the list.
+#               This directive clears the "nopass" flag (see later).
+#  <<password>  Remove this password from the list of valid passwords.
+#  nopass       All the set passwords of the user are removed, and the user
+#               is flagged as requiring no password: it means that every
+#               password will work against this user. If this directive is
+#               used for the default user, every new connection will be
+#               immediately authenticated with the default user without
+#               any explicit AUTH command required. Note that the "resetpass"
+#               directive will clear this condition.
+#  resetpass    Flush the list of allowed passwords. Moreover removes the
+#               "nopass" status. After "resetpass" the user has no associated
+#               passwords and there is no way to authenticate without adding
+#               some password (or setting it as "nopass" later).
+#  reset        Performs the following actions: resetpass, resetkeys, off,
+#               -@all. The user returns to the same state it has immediately
+#               after its creation.
+#
+# ACL rules can be specified in any order: for instance you can start with
+# passwords, then flags, or key patterns. However note that the additive
+# and subtractive rules will CHANGE MEANING depending on the ordering.
+# For instance see the following example:
+#
+#   user alice on +@all -SHUTDOWN ~* >somepassword
+#
+# This will allow "alice" to use all the commands with the exception of the
+# SHUTDOWN command, since +@all added all the commands to the set of the commands
+# alice can use, and later SHUTDOWN was removed. However if we invert the order
+# of two ACL rules the result will be different:
+#
+#   user alice on -SHUTDOWN +@all ~* >somepassword
+#
+# Now SHUTDOWN was removed when alice had yet no commands in the set of allowed
+# commands, later all the commands are added, so the user will be able to
+# execute everything.
+#
+# Basically ACL rules are processed left-to-right.
+#
```

Comment:
```suggestion
```

Reply:
thanks @itamarhaber , I folllowed alll your changes except i think it is also worth to mention user can also refer to redis.conf template to configure acl for sentinel. how do you think on this?  thanks!.